.github/workflows/ci.yml

```diff
@@ -75,7 +75,8 @@ jobs:
           make -j 4
           cd ..
           python3 run_tests.py regression.suite \
-            --exe build/src/artemis \
+            --save_build \
+            --reuse_build \
```

Comment:
Not sure I understand this change -- we just built the code with `make -j 4` on line 75, so we don't want to rebuild the code here

Reply:
This was to just not go down the `--exe` decision paths for this testing since I had changed some things

Reply:
We skip the build step here since I added `--reuse_build`, so I think it's okay.

Reply:
Ah yes you're right, I hadn't thought through the interaction of `--exe` and `--reuse_build`. Indeed, the code is only built once on the cpu ci.